GEM/lua/entities/gem_emulator/cl_init.lua

```diff
@@ -455,12 +455,141 @@ end
 -- Name: Think
 -- Desc: Handles the touch screen among other things
 ----------------------------------------------------------------------
-local keys = {}
-for k,v in pairs( _E ) do
-	if k:sub(1,4) == "KEY_" then
-		keys[k] = v
-	end
-end
+local keys = {

```

Comment:
The code above this that you erased already worked fine, you just had to change _E to _G